### DISCUSSION TITLE
VS构建执行失败

### DISCUSSION THREAD

**Discussion by smzkpy** [ywmoyue/biliuwp-lite#249] [Q&A]
如图，前面所有流程全部完成，就是最后一个一直报错，卡了很久。不知道问题在哪？
![QQ截图20230807150738](https://github.com/ywmoyue/biliuwp-lite/assets/33112232/70eaf67b-ec89-42a5-aa65-7a1f1f16cee1)

**Comment by ywmoyue** [comment]
缺少Windows SDK，使用VisualStudio Installer进行安装
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/2b18834c-7489-435e-9892-d7af09421588)

**Reply by smzkpy** [reply]
没有用，按照上面的选项都装了，还是这个问题。会不会是没有装C盘的问题（C盘只有5G根本装不了。。。。。。
![QQ截图20230807211836](https://github.com/ywmoyue/biliuwp-lite/assets/33112232/b9e0bb3d-4265-4abe-9b91-5729b86ffb5f)
![QQ截图20230807211855](https://github.com/ywmoyue/biliuwp-lite/assets/33112232/5871586b-6b76-4859-9f31-603d31ee429f)

**Comment by ywmoyue** [comment]
看是否和这个是同一个问题：https://github.com/ywmoyue/biliuwp-lite/discussions/206

**Reply by smzkpy** [reply]
问题很类似，但还是有点不一样。在提问之前我就装了17763版本的Windows SDK，但还是没有效果，不如说装了之后会提示你版本过老不能使用，说明不是这个问题。
我在想会不会是因为用的是64位版本但是32位版本没有所以无法被调用（上网找了一天自己猜的
或者还有一种可能，这个代码就不能用VS2022使用，必须要用老版本才能调用。要真是这样就乐了（血压上升.jpg

**Reply by ywmoyue** [reply]
我的VS就是2022的

**Reply by smzkpy** [reply]
\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2634,5): error MSB3774: 找不到 SDK“WindowsDesktop, Version=10.0.18362.0”
一直都是这个问题。系统盘里面只有22000的包，就是没有18632的包，感觉只要把这个问题解决就没有什么问题了
![QQ截图20230809145702](https://github.com/ywmoyue/biliuwp-lite/assets/33112232/9576d04a-fb1a-4060-8fce-d70838c52cfa)
![QQ截图20230809145636](https://github.com/ywmoyue/biliuwp-lite/assets/33112232/7e49f30e-780c-496e-87ce-b50b3d96f2e8)
